### ISSUE TITLE
GPU resources for training

### ISSUE THREAD

**Issue by Johnx69** [gaoyuezhou/dino_wm#1]
This is an outstanding piece of work! How many GPUs are required for training?

Thank you!!!

**Comment by gaoyuezhou** [comment]
Hi, 

Thank you for your interest in our work! For the results in the paper, each model can be trained on a single GPU (e.g. NVIDIA A6000), though it may take 1 to 2 days for environments with large datasets. However, our code supports Distributed Data Parallel (DDP), allowing for much faster training depending on the number of GPUs available.

In our experiments, we have trained with up to 32 NVIDIA H100 GPUs in parallel, which significantly speeds up training—large datasets that take a day on one GPU can be trained in just 1 to 2 hours with more GPUs. 

Let me know if you have more questions!

Best,
Gaoyue

**Comment by ykx3** [comment]
Hi @gaoyuezhou,

I've been following this thread and found the information about the GPU setup very helpful.

I have a follow-up question regarding your comment on scaling the training to 32 H100s using DDP: How did you adjust the learning rate when increasing the number of GPUs?

Since DDP scales the total batch size, I'm curious if you applied a strategy like the "Linear Scaling Rule" (scaling the LR linearly with the batch size), or if it required a different approach to tuning?

Thanks for your time!
